packages/editor/src/components/editor/properties/fields/browser/variable-tree-data.ts

```diff
@@ -0,0 +1,45 @@
+import type { Variable, VariableInfo } from '@axonivy/form-editor-protocol';
+import type { BrowserNode } from '@axonivy/ui-components';
+import { IvyIcons } from '@axonivy/ui-icons';
+
+export interface VariableTreeData extends BrowserNode<Variable> {
+  isLoaded: boolean;
```

Comment:
This properties are already defined in BrowserNode since your last PR. Why do you need to redefine them here?

Reply:
I think it is indeed no longer needed